src/pages/jaehyun/Login/InputBox/InputBox.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+class InputBox extends React.Component {
+  // eslint-disable-next-line no-useless-constructor
+  constructor(props) {
+    super(props);
+  }
+  // inputHandler = e => {
+  //   const { name, value } = e.target;
+  //   this.props.recivedValue(name, value, this.inputValidation(name, value));
+  // };
+
+  // inputValidation(name, value) {
+  //   let disable;
+  //   if (name === 'userName') {
+  //     disable = value.indexOf('@') ? false : true;
+  //     return disable;
+  //   } else if (name === 'password') {
+  //     disable = value.length > 5 ? false : true;
+  //     return disable;
+  //   }
+  // }
+
+  render() {
+    return (
+      <input
+        name={this.props.name}
+        type={this.props.type}
+        placeholder={this.props.placeholder}
+        onChange={this.props.recivedValue}
+      />
+    );
```

Comment:
```suggestion
    const {name, type, placeholder, recivedValue} = this.props;
    return (
      <input
        name={name}
        type={type}
        placeholder={placeholder}
        onChange={recivedValue}
      />
    );
```
구조분해할당해서 쓰면 좀 더 깔끔하겠네요